config/aic.php

```diff
@@ -39,4 +39,28 @@
     'show_hours_in_footer' => (bool) env('SHOW_HOURS_IN_FOOTER', false),
     'disable_captcha' => (bool) env('DISABLE_CAPTCHA', false),
     'show_default_related_items' => (bool) env('SHOW_DEFAULT_RELATED_ITEMS', true),
+
+    // Branding
+    'branding' => [
+        'colors' => [
```

Comment:
Call this `digital_publication_colors`, as we can use this to collate colors for other sub-brands on the site as well.

Reply:
I made it `['branding' => ['digital_publications' => ['colors' => [...]]]` or `config('aic.branding.digital_publications.colors')` in case there are other digital-publication-brand-specific things besides colors we might want to include.